README.md

```diff
@@ -18,7 +18,6 @@
         <ul>
             <li><strong><a href="https://github.com/skanda890/skanda890">skanda890</a></strong>: Configuration files and two beautiful README files for my GitHub profile.
             <li><strong><a href="https://github.com/skanda890/CodePark">CodePark</a></strong>: Described as a personal repository, CodePark contains machine languages respective to the branches.</li>
-            <li><strong><a href="https://github.com/skanda890/Hackbook">Hackbook</a></strong>: An Application for Tamil Nadu Police Cyber Cell Department according to their problem statement.</li>
```

Comment:
> **question (documentation):** Removed project from list - intentional?
> 
> The Hackbook project has been removed from the list. Was this intentional? If so, you might want to consider mentioning any new projects that have replaced it or explaining why it's no longer listed.

Hackbook was removed because it no longer exists.